schema/artwork.js

```diff
@@ -87,11 +74,19 @@ let ArtworkType = new GraphQLObjectType({
         type: new GraphQLObjectType({
           name: 'dimensions',
           fields: {
-            in: { type: GraphQLString },
-            cm: { type: GraphQLString }
+            in: {
+              type: GraphQLString
+            },
+            cm: {
+              type: GraphQLString
+            }
           }
         })
       },
+      image: {
+        type: Image.type,
+        resolve: ({ images }) => _.findWhere(images, { is_default: true }) || _.first(images)
+      },
```

Comment:
I think just naming this `image` is better as it's more in line with the rest of the schema